build/Dockerfile-ubuntu-18.04-go1.20.6-solc0.8.13

```diff
@@ -26,4 +26,4 @@ RUN mkdir /usr/bin/solc && wget -q https://github.com/ethereum/solidity/releases
 ENV GOPATH /go
 ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
 RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
-RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.52.0
\ No newline at end of file
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.24.0
```

Comment:
Can we use v1.52.0 linter? I think latest linter is better.

Reply:
Okay, I will rollback to 1.52 but you will need to fix some lint in here: https://app.circleci.com/pipelines/github/klaytn/klaytn/10636/workflows/8c7fc54b-3c9f-42ef-9c29-d78653991e63/jobs/56668